components/homePage/Banner.tsx

```diff
@@ -1,12 +1,14 @@
 "use client";
-import { Box, Button, Grid, Typography } from "@mui/material";
```

Comment:
As a good practice, you should not remove this whole import and add it again with new imports.
Instead you should only add Modal and DialogContent to existing import,